src/inverseRobot.js

```diff
@@ -7,7 +7,16 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const INVERSE_KEYS_AND_VALUES = {};
```

Comment:
CAPITAL_SNAKE_CASE should be used only for global constants that will never mutate. (e.g. `const PI = 3.14151...`).

Here you should regularPascalCase. Check out [mates naming convention](https://github.com/mate-academy/style-guides/blob/master/javascript.md#23-naming-conventions).